### ISSUE TITLE
Crash using transmutation table after alchemical tome

### ISSUE THREAD

**Issue by bwinston** [sinkillerj/ProjectE#503]
After "burning" the Alchemical Tome in the Transmutation Table, whenever I attempt to search the table I end up crashing out. Note that the server on which I am playing has around 270 mods installed, so my thought is that there are so many items on the server, it is unable to search all of them properly. To further back this up, while I haven't tried every letter, starting to type in anything starting with the letter i doesn't crash, but letters such as A and R do. I've attached the crash report below.

https://www.dropbox.com/s/k7o0gpuwz2y94qw/crash-2015-02-17_13.06.57-client.txt?dl=0

**Comment by MaPePeR** [comment]
This is supposed to be fixed in the recent version and also a duplicate of at least one other issue.

Are you using the most recent version (yesterday)

**Comment by bwinston** [comment]
Ok, no we're not using the most recent version. We're waiting on the RR update to patch (which won't be for a while).

**Comment by badoquealex** [comment]
I have the same issue, after burning the Alchemical Tome, when i search certain items, ftb crash. No crash when i enter "ge" but when I press "m", crash...
